### ISSUE TITLE
jjjchens235/bing-rewards got disabled

### ISSUE THREAD

**Issue by Auncaughbove17** [jack-mil/bing-rewards#16]
just fyi, https://github.com/jjjchens235/bing-rewards and all of the forks got disabled. You might want to consider moving to gitlab.
I assume they care less, considering they are not owned by Microsoft afaik

**Comment by jack-mil** [comment]
Thanks for the heads-up. I haven't been following all the forks, are there additions being made? I would accept PRs if people are still using this. 
Do you know if GitHub disabled the repo, or if the user simply deleted the fork? 
Some of the others seem to still exist.

**Comment by Auncaughbove17** [comment]
I believe it is completely independent, it doesn't use PyAutoGUI. But it serves the same purpose.
I went to web.archive.org and got a list of all forks. All of the forks that I checked are also disabled. 
Also, the message when loading the github url it states:
> Access to this repository has been disabled by GitHub Staff due to a violation of GitHub's terms of service.

**Comment by jack-mil** [comment]
Oh my bad, this is a different project with the same name as mine? I looked into it. Looks they they were using a more complicated headless method using selenium. I believe I saw a few projects using this back before I made my script. My goal was to create a simpler semi-automated method as an alternative. If they come for me, I'll move to Gitlab, but the tool will still be available on PyPi.

Again, thanks for the heads-up. I need to revisit this project and give it some TLC, as I haven't tested or used it in a while and there are some issues. Maybe I'll have time for that soon.
